fsm_admin/mixins.py

```diff
@@ -48,6 +48,8 @@ class FSMTransitionMixin(object):
     fsm_field = ['state']
     change_form_template = 'fsm_admin/change_form.html'
     default_disallow_transition = not getattr(settings, 'FSM_ADMIN_FORCE_PERMIT', False)
+    success_message_template = _('%(obj)s successfully set to %(new_state)s')
+    error_message_template = _('Error! %(obj)s failed to %(transition)s')
```

Comment:
Would be nice to have a trailing dot here by default (since it is a whole sentence), but this might cause translations to become outdated etc, so maybe better to keep it as before by default.
(However the missing dot would be the only reason to override/customize it for me)